store/src/commonMain/kotlin/com/dropbox/external/store5/impl/MemoryLruCache.kt

```diff
@@ -0,0 +1,128 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package com.dropbox.external.store5.impl
+
+import com.dropbox.external.store5.Persister
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.sync.Mutex
+
+/**
+ * Thread-safe LRU cache implementation.
+ */
+class MemoryLruCache(private val maxSize: Int) : Persister<String> {
```

Comment:
Is this implementation meant as a replacement for [cache](https://github.com/MobileNativeFoundation/Store/tree/main/cache) ?

Reply:
No it will not replace existing Cache. In future diff I will add bindings to it https://github.com/MobileNativeFoundation/Store/issues/447

Reply:
Existing cache rocks. This is just meant to be another option

Reply:
Feel free to reopen if other concerns! Marking as resolved for now